tensorflow_recommenders_addons/dynamic_embedding/core/ops/cuckoo_hashtable_ops.cc

```diff
@@ -270,6 +277,13 @@ REGISTER_OP(PREFIX_OP_NAME(CuckooHashTableImport))
       return Status::OK();
     });
 
+REGISTER_OP(PREFIX_OP_NAME(CuckooHashTableImportFromFile))
+    .Input("table_handle: resource")
+    .Input("filepath: string")
+    .Attr("key_dtype: type")
+    .Attr("value_dtype: type")
+    .Attr("buffer_size: int >= 1");
```

Comment:
`int` could be overflow

Reply:
It is dialect in tf op_kernel, int32,int64 are included